src/com/cdd/bao/axioms/AxiomVocab.java

```diff
@@ -64,8 +64,8 @@ public static Type valueOf(int rawVal)
 
 	public static class Term
 	{
-		public String valueURI;
-		public boolean wholeBranch;
+		public String valueURI = null;
+		public boolean wholeBranch = false;
```

Comment:
These are the default values.

Reply:
Setting to null is a "clue" that the constructor is likely to just leave it that way... whereas leaving it un-initialised implies that the constructor is going to make the call explicitly.